client/scss/components/forms/_nested-panel.scss

```diff
@@ -0,0 +1,86 @@
+$header-icon-size-sm: theme('spacing.5');
+$header-button-size-sm: theme('spacing.8');
+$header-gap: theme('spacing.1');
+$panel-x-offset: calc($header-button-size-sm / 2 + $header-gap);
+
+/**
+ * Panel styles shared between StreamField and InlinePanel, for
+ * repeating collapsible panels which can be reordered.
+ * Top-level and nested panels have guiding borders to show their start and end,
+ * as well as indentation for nested panels.
+ */
+
+.w-panel--nested {
+  @include guide-line-active();
+
+  .w-panel__divider {
+    @include guide-line-horizontal();
+    // Slightly nicer text alignment.
+    margin-top: 1px;
+  }
+
+  .w-panel {
+    @include guide-line-nested();
+    // Nesting for nested panels.
+    margin-inline-start: theme('spacing.7');
+    margin-bottom: theme('spacing.6');
+  }
+
+  // Tighter spacing for nested panel headers so their icons align
+  // with parent panels’ guiding line.
+  .w-panel .w-panel__header {
+    gap: 0;
+    transform: translateX(calc(-1 * theme('spacing.5')));
+
+    @include media-breakpoint-up(sm) {
+      transform: translateX(theme('spacing.1'));
+    }
+  }
+
+  .w-panel__content {
+    @include guide-line-vertical();
+    // Center the vertical line.
+    margin-inline-start: calc(-1 * theme('spacing.3'));
+    padding-inline-start: theme('spacing.3');
+
+    @include media-breakpoint-up(sm) {
+      margin-inline-start: calc(-1 * $panel-x-offset);
+      padding-inline-start: $panel-x-offset;
+    }
+  }
+
+  .w-panel__anchor {
+    background-color: $color-white;
+  }
+
+  .w-panel .w-panel__anchor--prefix {
+    display: inline-grid;
+  }
+
+  .w-panel .w-panel__anchor--suffix {
+    display: none;
+  }
```

Comment:
Might be good to add a comment why we are hiding this. I assume it's because we have an alternative presentation of the same element as prefix on a different breakpoint. 

Otherwise it might be a bit confusing.

Reply:
👍 I’ve refactored this code so it’s clearer which styles are specifically for nested panels, and why we’re doing this in particular.